index.js

```diff
@@ -98,23 +102,24 @@ class ClientServiceBase {
    * @return {Promise<ClientServiceResponse>}
    */
   async registerCertificate() {
-    const request = new CertificateRegistrationRequestBuilder(
+    const builder = new CertificateRegistrationRequestBuilder(
```

Comment:
I'm wondering why `build()` is now defined with `async` and `await` for the `build()` ?
I'm not sure about the difference of the current way and the old way.

Also, is this anything to do with these web crypt changes ?

Reply:
The reason to use async in the build functions of all builders is that the WebCryptoAPI `sutle.sign` is an async function.
The build functions will serialize JavaScript object into gRPC requests and then sign the raw data.
```
function build() {
   ...
   ...
   let signature = this.signer.sign();

   return request;
}
```

The elliptic and jsrsasign libraries use a synchronized way to implement `sign()`, so that the `build` function can be synchronized or asynchronized.

However, Web Crypto API uses an asynchronized way to implement its sign function.
Therefore, in the `build` implementation, we can either choose:
1. Add a new argument for the build functions to get callback function from the upper layer. 
```
async function build() {
    ...
    ...
    let signature = await this.signer.sign(); // this is now probably an async function because of WebCryptoAPI
}
```

Or,
2. Change the build function from synchronized to asynchronized.
```
function build(callback) {
    ...
    ...
    this.signer.sign().then(
        signature => callback(signature)
    ).catch(
    );
}
```

I chose 1 because the upper layer of builders,
for example, `ClientService.registerCertificate()` is actually been implemented in asynchorized function already. My change will not impact any interfaces.